libs/application/templates/new-primary-school/src/fields/RelativesTableRepeater/index.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  FieldBaseProps,
+  FieldComponents,
+  FieldTypes,
+  YES,
+} from '@island.is/application/types'
+import {
+  formatPhoneNumber,
+  removeCountryCode,
+} from '@island.is/application/ui-components'
+import { TableRepeaterFormField } from '@island.is/application/ui-fields'
+import { format as formatKennitala } from 'kennitala'
+import React, { FC } from 'react'
+import { useFriggOptions } from '../../hooks/useFriggOptions'
+import { OptionsType } from '../../lib/constants'
+import { newPrimarySchoolMessages } from '../../lib/messages'
+import { getSelectedOptionLabel } from '../../lib/newPrimarySchoolUtils'
+
+const RelativesTableRepeater: FC<
+  React.PropsWithChildren<FieldBaseProps>
+> = ({ error, field, application }) => {
+  const { title } = field
+
+  const relationFriggOptions = useFriggOptions(OptionsType.RELATION)
+
+  return (
+    <TableRepeaterFormField
+      application={application}
+      error={error}
+      field={{
+        type: FieldTypes.TABLE_REPEATER,
+        component: FieldComponents.TABLE_REPEATER,
+        children: undefined,
+        id: field.id,
```

Comment:
Might as well destructure 'id'
```suggestion
  const { id, title } = field

  const relationFriggOptions = useFriggOptions(OptionsType.RELATION)

  return (
    <TableRepeaterFormField
      application={application}
      error={error}
      field={{
        type: FieldTypes.TABLE_REPEATER,
        component: FieldComponents.TABLE_REPEATER,
        children: undefined,
        id,
```